shared.jq

```diff
@@ -0,0 +1,23 @@
+def apt_get_dist_clean:
+	if [
+		# only suites with APT 2.7.8+ have "apt-get dist-clean"
+		# https://tracker.debian.org/news/1492892/accepted-apt-278-source-into-unstable/
+
+		"trixie", # TODO once 2.7.8 migrates to testing (and images are rebuilt), this should be removed!
+		"bookworm",
+		"bullseye",
+		"buster",
+
+		"noble", # TODO once 2.7.8+ makes it into devel (and images are rebuilt), this should be removed!
```

Comment:
https://launchpad.net/ubuntu/noble/+source/apt :eyes:

(APT 2.7.9 is in proposed now, so hopefully will be in the next `ubuntu:noble` image update :muscle:)

Reply:
Not https://github.com/docker-library/official-images/pull/16059, but will hold out hope for the _next_ one (it's still in `proposed` so this isn't unexpected) :+1: